finish/src/main/java/io/openliberty/guides/bulkhead/global/eBank/microservices/Utils.java

```diff
@@ -8,9 +8,9 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package global.eBank.microservices;
+package io.openliberty.guides.bulkhead.global.eBank.microservices;
```

Comment:
The chat input and button should have "For demo only" instead of "For demo only and will not do anything".